modules/ting_search/ting_search.module

```diff
@@ -944,19 +1012,7 @@ function ting_search_submit($form, &$form_state) {
   $keys = $form_state['values'][$form_id];
   $fields = array();
 
-  if (!empty($form_state['values']['material_type'])) {
-    if (!empty($keys)) {
-      $controls['facets'] = array('facet.type:' . $form_state['values']['material_type']);
-    }
-    // If there's no search string, add the facet.type search to the actual
-    // string, or the search would return no results.
-    else {
-      $keys = 'facet.type="' . $form_state['values']['material_type'] . '"';
-      $controls['sort'] = 'date_descending';
-    }
-
-  }
-
+  // TODO BBS-SAL BBS-217: Convert this to SAL.
```

Comment:
That would be nice 👍

Reply:
A ticket should be created for this.

We have not had the time to migrate handling of expanded search in time. The current situation will not harm OpenSearch libraries.